tensorflow/tools/api/golden/v2/tensorflow.raw_ops.pbtxt

```diff
@@ -1248,6 +1248,10 @@ tf_module {
     name: "DrawBoundingBoxesV2"
     argspec: "args=[\'images\', \'boxes\', \'colors\', \'name\'], varargs=None, keywords=None, defaults=[\'None\'], "
   }
+  member_method {
+    name: "DrawBoundingBoxesV3"
+    argspec: "args=[\'images\', \'boxes\', \'colors\', \'thickness\', \'name\'], varargs=None, keywords=None, defaults=[\'1\', \'None\'], "
```

Comment:
Do you know where this default "1" is coming from? The one in the Python API makes sense since that came from the argument list. But I don't see a default on the OpDef, and didn't think you could have defaults for tensor inputs.

Reply:
I had taken inspiration from how `ExperimentalThreadPoolHandle` member was defined in the same file. Honestly I am not too sure on this either, I just assumed that this is how it had to be done. Do let me know if you think this should be done some other way.

Reply:
ExperimentalThreadPoolHandle mostly has attributes, not inputs. Attributes can definitely have default values.

The command in this readme updates the pbtxt files automatically: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/api/tests

I'd just use whatever that gives you, since the test will fail otherwise.